x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/external/action/SingleInputSenderExecutableActionTests.java

```diff
@@ -61,25 +61,11 @@ public void testOneInputIsValid() {
         assertTrue("Test failed to call listener.", testRan.get());
     }
 
-    public void testInvalidInputType() {
```

Comment:
We can no longer check for the type of the input since it can be from completion or text embedding.